src/Dev/TaskRunner.php

```diff
@@ -207,4 +236,24 @@ protected function addCssToHeader($header)
 
         return $header;
     }
+
+    public function canInit(): bool
+    {
+        if ($this->canViewAllTasks()) {
+            return true;
+        }
+        return count($this->getTaskList()) > 0;
+    }
+    
+    public function providePermissions(): array
+    {
+        return [
+            'BUILDTASK_CAN_RUN' => [
+                'name' => _t(__CLASS__ . '.BUILDTASK_CAN_RUN_DESCRIPTION', 'Can view and run Build Tasks'),
+                'help' => _t(__CLASS__ . '.BUILDTASK_CAN_RUN_HELP', 'Can view and run Build Tasks via the endpoint (/dev/tasks).'),
```

Comment:
```suggestion
                'name' => _t(__CLASS__ . '.BUILDTASK_CAN_RUN_DESCRIPTION', 'Can view and run /dev/tasks'),
                'help' => _t(__CLASS__ . '.BUILDTASK_CAN_RUN_HELP', 'Can view and run Build Tasks (/dev/tasks).'),
```
Consistent with the others

Reply:
I'm also now seeing that this permission overrides canView checks for the tasks - the language should be updated to reflect that. It's not clear right now that:
1. Some people may be able to view build tasks without this permission
2. Anyone with this permission will bypass any explicit canView checks on `BuildTask`s